handler/setup.go

```diff
@@ -200,6 +202,22 @@ func getDockerSockVolume() *spec.Volume {
 	}
 }
 
+func appendConfigVolume(volumes []*spec.Volume) []*spec.Volume {
+	loadedConfig, _ := config.Load()
```

Comment:
Pass in this config to the HandleSetup / HandleStep handlers when they are loaded, so that we don't have to load from env variables for each call